install_homeassistant.sh

```diff
@@ -1,8 +1,19 @@
 #!/bin/sh
```

Comment:
Ok, it's not as difficult as I first thought.  I've just pushed a new change in so that the install_homeassistant.sh script when run standalone acts as if it were the old installer.  Bonus: it still fits in the hassbian-config mechanism so it can be used from there too.
This also fixes the install_homeassistant.sh script's hashbang which somehow hadn't been edited to use bash, but /bin/sh - which precluded the whole lot working.